src/test/java/com/amplitude/util/EventsGenerator.java

```diff
@@ -0,0 +1,35 @@
+package com.amplitude.util;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.UUID;
+
+import com.amplitude.Event;
+
+public class EventsGenerator {
+  public static List<Event> generateEvents(int eventCount) {
+    return generateEvents(eventCount, 1, 1);
+  }
+
+  public static List<Event> generateEvents(int eventCount, int userCount, int deviceCount) {
```

Comment:
Rename these parameters to `userIdLength` and `deviceIdLength`? A bit confusing to me as I thought multiple users were generated as some sort of event/user/group property.

Reply:
how about `userIdCount` or `deviceIdCount`, `userIdLength` or `deviceIdLength` is confusing me to consider it is about the length of user id or device id. The goal for this helper method is to just generate events for multiple users or devices.

Reply:
Yeah, userCount to me sounds like a number of unique user IDs or something.

Reply:
yeah, that's actually the goal for this, generate event for various users.